inductiva/utils/format_utils.py

```diff
@@ -113,4 +152,18 @@ def get_tabular_str(tabular_data: Union[Mapping[str, Iterable[Any]],
     """
 
     data, headers = get_tabular_data(tabular_data, headers, formatters)
-    return tabulate(data, headers=headers, missingval="n/a")
+
+    header_formatters = header_formatters or []
+    for formatter in header_formatters:
+        headers = [formatter(header) for header in headers]
+    if indentation_level is not None:
+        headers = [" " * indentation_level + header for header in headers]
+
+    table = tabulate.tabulate(data,
+                              headers=headers,
+                              missingval="n/a",
+                              tablefmt="plain")
+    if indentation_level is not None:
+        table = _table_indenter(table, indentation_level)
```

Comment:
Are headers indented twice? Or you first need to generate the table and only afterwards you can indent the body?

Reply:
Solved